### PR TITLE
Move net46 to net461

### PR DESCRIPTION
**Pull request by SimonCropp**
and simplify value tuple code

### CODE REVIEW

**Comment by danielchalmers** [comment]
excuse me if this isn't relevant, just wanted to add that 4.6.1 is out of support and 4.6.2 is the earliest supported version, so could skip to that - https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework

**Comment by nblumhardt** [comment]
Thanks @danielchalmers; some discussion going on in #1820

File: src/Serilog/Serilog.csproj [hunk 1]
```diff
@@ -13,34 +13,34 @@
...
+  <ItemGroup>
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+  </ItemGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net47' ">
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <DefineConstants>$(DefineConstants);FEATURE_VALUETUPLE</DefineConstants>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
```
Comment (sungam3r) [inline src/Serilog/Serilog.csproj:27]:
```suggestion
```

File: src/Serilog/Serilog.csproj [hunk 2]
```diff
@@ -13,34 +13,34 @@
...
-    <DefineConstants>$(DefineConstants);FEATURE_VALUETUPLE</DefineConstants>
   </PropertyGroup>
```
Comment (sungam3r) [inline src/Serilog/Serilog.csproj:28]:
```suggestion
```

File: src/Serilog/Serilog.csproj [hunk 3]
```diff
@@ -13,34 +13,34 @@
...
```
Comment (sungam3r) [inline src/Serilog/Serilog.csproj:29]:
```suggestion
```

**Review by sungam3r — APPROVED** [review]

**Pull request merged**